src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
`Health is 100 by default.`
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```